src/Config/GraphQLRuntimeOptions.cs

```diff
@@ -0,0 +1,6 @@
+namespace Azure.DataApiBuilder.Config;
+
+public record GraphQLRuntimeOptions(bool Enabled = true, string Path = GraphQLRuntimeOptions.DEFAULT_PATH, bool AllowIntrospection = true)
```

Comment:
I think we can move files such as this one, HostMode.cs, HostOptions.cs, JwtOptions.cs, etc inside a directory called RuntimeSettings or RuntimeOptions as all these options come under the runtime section in the config file.

Reply:
I was going to create a folder like `ObjectModel` and move all the types in there, but VS was having problems doing the refactor without a lot of manual intervention. I'll update my build and try again.